### ISSUE TITLE
Is there better way for field with argument

### ISSUE THREAD

**Issue by rikkei-minh** [99designs/gqlgen#1378]
### What happened?
I got graphql.schema where model field got agurment.
After running gqlgen generate command, I can not see any sign of resolver for friends.
All friends-related generated content that I got:

```
// models_gen.go
type User struct {
    ID string
    Friends []*Friend
}

// generated.go
...
type ComplexityRoot struct {
    ...
    Friends func(childComplexity int, count *int) int
    ...
}

func (ec *executionContext) field_User_friends_args(...) {
    ...
}
```
I found some ways to work-around with this:
① Following this guide [https://gqlgen.com/reference/resolvers/] to bind to method name. But I don't prefer this way since i have to define User struct myself. I would like to use graphql.schema and generate code from the schema.
② Using `for _, f := range graphql.CollectFieldsCtx(ctx, nil)` to get the argument value and handle it somewhere in Users resolver. This way will be bad idea if my argument is changed.
③ Add something like UserResolver() into ` type ResolverRoot interface`. In this way I have to edit content of generated.go file which was generated by gqlgen?
### What did you expect?
Is there any better solution(s) than above?
I read somewhere on issue list and found a similar issue where the contributor said that gqlgen can generate resolver for this case but ...

### Minimal graphql.schema and models to reproduce

Here is my schema
```
type User {
    id: ID!
    friends(count: Int) [Friend]
}

type Query {
    users(): [User]
}
```

### versions
 - `gqlgen version`? v0.13.0
 - `go version`? go1.15.2 windows/amd64
 - dep or go modules? module

**Comment by frederikhors** [comment]
Maybe you need to use `resolver: true` in your gelgen.yml file.

**Comment by rikkei-minh** [comment]
@frederikhors, thank you very much. I'm trying this setting now.

**Comment by rikkei-minh** [comment]
@frederikhors , I applied your guide and it's worked like a charm.
Thank you very much.
In order to keep it as a memo for the others that have the same problem like myself, I note it as below:
1. I added following setting into gqlgen.yml
```
models
  ...
  User:
    fields:
      friends:
        resolver: true
```
2. I ran gqlgen generate command. Then gqlgen generated for me the resolver for the friends inside User:
`func (r *userResolver) Friends(...)`
So I can implement filter logic for friends of User.
